app/settings/index.js

```diff
@@ -0,0 +1,7 @@
+import {environment} from './environments'
+
+export const SERVER_API = {
```

Comment:
Would it make sense to return a settings object ?

```
export default {
  development: {SERVER_API: '...'},
  staging: {SERVER_API: '...'},
  production: {SERVER_API: '...'}
}[environment]
```

Reply:
what about if there are settings that are the same across all environments?

Reply:
set it as a variable above the exported object? Or have a defaults object that is merged into each env object?

I'm cool either way, just thought it might get messy with many settings variables:

```
export const VAL_1 = {
  development: 'http://localhost:8000',
  staging: '[STAGING_URL]',
  production: '[PRODUCTION_URL]'
}[environment]

export const VAL_2 = {
  development: 'http://localhost:8000',
  staging: '[STAGING_URL]',
  production: '[PRODUCTION_URL]'
}[environment]

export const VAL_3 = {
  development: 'http://localhost:8000',
  staging: '[STAGING_URL]',
  production: '[PRODUCTION_URL]'
}[environment]

export const VAL_4 = {
  development: 'http://localhost:8000',
  staging: '[STAGING_URL]',
  production: '[PRODUCTION_URL]'
}[environment]

export const VAL_5 = {
  development: 'http://localhost:8000',
  staging: '[STAGING_URL]',
  production: '[PRODUCTION_URL]'
}[environment]
```

Reply:
I agree the above example is too messy. I think it depends how many settings we have that change based on the environment. 
I tend to prefer
```
export const VAL_5 = {
  development: 'http://localhost:8000',
  staging: '[STAGING_URL]',
  production: '[PRODUCTION_URL]'
}[environment]
```
because normally I go to settings looking for a certain setting name, and don't like to jump around different environments for the same setting